docs/GraFx-Studio/guides/output/settings/index.md

```diff
@@ -1,11 +1,108 @@
----
-tags:
-  - unfinished
----
+# Create Output Settings
 
-# Output Settings
+[What are output settings?](/GraFx-Studio/concepts/output-settings/)
 
-Placeholder to describe where and how to create output settings.
-[See also description](https://chilipublishintranet.atlassian.net/wiki/spaces/dev/pages/3806691367/GraFx+Studio+MVP+Animation+export+settings){target="_blank"}
+## Create output settings
 
-(link will be removed, once content is ready)
\ No newline at end of file
+In the GraFx Studio application, under manage, you can manage "Output settings"
+
+To add a setting click the "+ create" button.
+
+![screenshot-full](os01.png)
+
+Give your setting a relevant name, and choose the output file format.
+
+![screenshot](os03.png)
+
+The created setting will appear in the list.
+
+![screenshot-full](os04.png)
+
+You can delete a setting, at the "..." menu at the right of the list.
+
+![screenshot](os06.png)
+
+Depending on the chosen output file type, there are different settings available.
+
+## Generic settings for all file types
+
+![screenshot-full](os14.png)
+
+### Name and description
+
+The name of your setting (1), is the name that will be shown in the [Studio UI](/GraFx-Studio/guides/create-projects/#customize-your-project).
+
+The description (2) will be shown as a tooltip, when hovering over the output setting in the Studio UI (4).
```

Comment:
The description is not shown as a tooltip but as a 2nd line in the dropdown in Studio UI. You can try it out by adding a description and you will see that the preview is updated on the fly.